parse/raw_data.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
i get `FileNotFoundError: [Errno 2] No such file or directory: 'project_raw/HDFS.log'
` when i run the 2nd cell